docs/src/main/asciidoc/doc-create-tutorial.adoc

```diff
@@ -180,7 +180,7 @@ Let's add the following code to that file:
 // tag::application[]
 package org.acme;
 
-import javax.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.ApplicationScoped;
```

Comment:
That is a 3.x content and won't work in 2.16.x

Reply:
@gastaldi & @gsmet - Thanks :+1: Fixed.